pkgs/development/libraries/science/math/cutensor/generic.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation {
 
   src = fetchurl {
     url = "https://developer.download.nvidia.com/compute/cutensor/${mostOfVersion}/local_installers/libcutensor-${stdenv.hostPlatform.parsed.kernel.name}-${stdenv.hostPlatform.parsed.cpu.name}-${version}.tar.gz";
-    inherit sha256;
+    inherit hash;
```

Comment:
Why `hash` vs `sha256`?

Reply:
Iiuc hash is preferred to sha256 as it is more future proof, but I don't remember the reference for that

Reply:
```
, # SRI hash.
  hash ? ""

, # Legacy ways of specifying the hash.
  outputHash ? ""
, outputHashAlgo ? ""
, md5 ? ""
, sha1 ? ""
, sha256 ? ""
, sha512 ? ""
```

Hmm, I guess

Reply:
yep, we are switching everywhere (though slowly) to SRI hashes.